packages/core/src/rules/specifyDlqOnEventBridgeRule/test.ts

```diff
@@ -0,0 +1,46 @@
+import { LambdaFunction } from 'aws-cdk-lib/aws-events-targets';
+import { Construct } from 'constructs';
+import {
+  DefaultEventBridgeRule,
+  DefaultFunction,
+  DefaultSqsQueue,
+} from '../../../tests/constructs';
+import { specifyDlqOnEventBridgeRule as SpecifyDlqOnEventBridgeRule } from './index';
+
+interface SpecifyDlqOnEventBridgeProps {
+  eventBridgeRuleHasTargetWithDlq: boolean;
+}
+
+export class SpecifyDlqOnEventBridge extends Construct {
+  static passTestCases: Record<string, SpecifyDlqOnEventBridgeProps> = {
+    'Defined dlq for target of EventBridge Rule': {
+      eventBridgeRuleHasTargetWithDlq: true,
```

Comment:
not super consistant with my naming on test from snsRedrivePolicy
hasDeadLetterQueue: true

But tbh don't know what's best and low logaf